### PR TITLE
feat: analytics

### PR DESCRIPTION
**Pull request by mshaaban0**
google analytics integration

### CODE REVIEW

**Comment by Khaledgarbaya** [comment]
@mshaaban0 I think we might need to send pageview events manually since this is a single page app

a few links:
- https://github.com/react-ga/react-ga/wiki/React-Router-v4-withTracker

File: src/App.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ import Home from './routes/Home';
 import Faq from './routes/Faq';
 import screenshot from './assets/screenshot.png';
 
+const GA_ID = 'UA-161843176-1';
```
Comment (freakyfelt) [inline src/App.js]: Can you add this to a `.env` file instead? https://create-react-app.dev/docs/adding-custom-environment-variables/#adding-development-environment-variables-in-env

Comment (mshaaban0) [reply src/App.js]: @freakyfelt Cool, do you think the `.env` should live in the git repo, we can add it to Netlify env variable, but I don't have access to Netlify.

Comment (freakyfelt) [reply src/App.js]: Given it's a client-visible variable it's probably fine for it to be committed, though if we add *any* other secret we'll want to revisit that

**Pull request merged**